src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalController.java

```diff
@@ -167,7 +167,7 @@ public ResponseEntity<Object> modifyRolesForExistingUserOfOrganisation(
         organisationIdentifierValidatorImpl.validateOrganisationExistsWithGivenOrgId(orgId);
 
         //Received request to update user roles of an organisation
-        return modifyRolesForUserOfOrganisation(userProfileUpdatedData, userId, Optional.of(origin));
+        return professionalUserService.modifyRolesForUser(userProfileUpdatedData, userId, Optional.of(origin));

```

Comment:
just a note. additional QA for internal controller endpoint will be required - as ticket is only for external endpoint update